compiler/rustc_target/src/spec/targets/i686_unknown_redox.rs

```diff
@@ -0,0 +1,27 @@
+use crate::spec::{base, Cc, LinkerFlavor, Lld, StackProbeType, Target};
+
+pub fn target() -> Target {
+    let mut base = base::redox::opts();
+    base.cpu = "pentiumpro".into();
```

Comment:
This makes i686_unknown_redox our only "i686" target that does not use pentium4 as baseline. Is that deliberate?  It is certainly very surprising.

Reply:
Ah no it is not alone, just some of the footnotes were wrong...

Reply:
So in total there are 2 i686 targets that do not use pentium4 as a baseline: this one and i686-unknown-hurd-gnu. I still think that's a bug, or was it explicitly discussed somewhere to deviate from our usual naming scheme? (And yes I know that naming scheme is technically wrong but we're not renaming a tier 1 target.)